src/Pages/Terms/Terms.scss

```
@@ -1 +1,147 @@
 @import "../../Styles/common.scss";
+
+
+
+.Terms {
+  .kakaoWrap { 
+    display: flex;
+    // justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 80px;
+
+
+    .kakaoHead {
+      img {
+        width: 85px;
+        height: 25px;
+      }
+    }
+    .kakaoContent {
+      margin: 40px auto;
+      padding: 58px 69px 65px;
+      border: 1px solid #e5e5e5;
+      border-radius: 2px;
+
+      h2 {
+        font-size: 24px;
+        font-weight: normal;
+        margin-bottom: 44px;
+      }
+
+      .termForm {
+        font-size: 15px;
+
+        .allBtn {
+          background: url(/Images/noneCheck.png) center center / 100% no-repeat;
+          width: 20px;
+          height: 20px;
+          margin-right: 8px;
+          outline: none;
+          border: 0;
+          cursor: pointer;
+        }
+        .AllCheck {
+          background: url(/Images/Check.png) center center / 100% no-repeat;
+          
+        }
+        // .AllCheck {
+        //   background: url(/Images/Check.png) center center / 100% no-repeat;
+        //   width: 20px;
+        //   height: 20px;
+        //   margin-right: 8px;
+        //   outline: none;
+        //   border: none;
+        // }
+
+        // .AllCheck {
+        //   background: url(/Images/Check.png) center center / 100% no-repeat;
+        //   width: 20px;
+        //   height: 20px;
+        //   margin-right: 8px;
+        //   outline: none;
+        //   border: none;
+        // }
+
+        .termsAll {
+          display: flex;
+          align-items: center;
+          cursor: pointer;
+
+          span {
+          font-weight: 800;
+          font-size: 17px;
+          color: #333;
+          }
+        }
+        .termsAllText {
+          padding: 10px 40px 20px 28px;
+          border-bottom: 1px solid #e5e5e5;
+          font-size: 12px;
+          color: #959595;
+        }
+        .termAgeOver {
+          display: flex;
+          align-items: center;
+          margin-top: 20px;
+          cursor: pointer;
+        }
+        .termsWrap {
+          display: flex;
+          align-items: center;
+          margin-top: 20px;
+          cursor: pointer;
+        }
+        .deskCheck {
+          padding: 10px 40px 0 28px;
+          font-size: 12px;
+          color: #959595;
+        }
+      }
+      .nextBtn {
+        margin-top: 40px;
+        .btn {
+          width: 100%;
+          height: 50px;
+          outline: none;
+          border: 0;
+          border-radius: 4px;
+          font-size: 15px;
+          color: #333;
+          background-color: #f6f6f6;
+          cursor: pointer;
+        }
+        .active {
+          background-color: #ffe500;
+        }
+        // .btn:hover {
+        //   background-color: #e7e7e7;
+        // }
+      }
+    }
+  }
+
+
+  .information {
+    padding: 20px 0 60px;
+    font-size: 12px;
+    .infoTop {
```

```suggestion
    font-size: 12px;
    
    .infoTop {
```
새로운 셀렉터 앞에서 한줄 띄워주세요

---

수정 완료했습니다 !